### PR TITLE
light colors working, fixed spot light cone angle, added initial support for directional lights (not spawning yet)

### PR DESCRIPTION
**Pull request by KaiserM21**

### CODE REVIEW

File: src/mods/liana_main.py [hunk 1]
```diff
@@ -1354,13 +1354,16 @@ def import_umap(settings: Settings, umap_data: dict, umap_name: str):
...
                             light_object.data.energy = (light_intensity*99.5*(1-cos(cone_angle/2)))/683
-                    if "LightColor" == prop_name:
-                        light_object.data.color = get_rgb_255(prop_value)[:-1]
-                    if "SourceRadius" == prop_name:
-                        if light_type == "SPOT":
-                            light_object.data.shadow_soft_size = prop_value * 0.01
-                        else:
-                            light_object.data.shadow_soft_size = prop_value * 0.1
+                    if light_type == "SUN":
+                        light_object.data.energy =(light_intensity*4*pi)/683
```
Comment (floxay) [inline src/mods/liana_main.py:1358]: Please add spaces. (and in the lines above as well, should've been done in #13)

File: src/mods/liana_main.py [hunk 2]
```diff
@@ -1354,13 +1354,16 @@ def import_umap(settings: Settings, umap_data: dict, umap_name: str):
...
+
+                if "LightColor" == prop_name:
```
Comment (floxay) [inline src/mods/liana_main.py:1360]: elif

File: src/mods/liana_main.py [hunk 3]
```diff
@@ -1354,13 +1354,16 @@ def import_umap(settings: Settings, umap_data: dict, umap_name: str):
...
+                    light_object.data.color = get_rgb_255(prop_value)
+                if "SourceRadius" == prop_name:
```
Comment (floxay) [inline src/mods/liana_main.py:1362]: elif

File: src/mods/liana_main.py [hunk 4]
```diff
@@ -1373,7 +1376,7 @@ def import_umap(settings: Settings, umap_data: dict, umap_name: str):
                     if "InnerConeAngle" == prop_name:
                         light_object.data.spot_blend = 1
                     if "OuterConeAngle" == prop_name:
-                        light_object.data.spot_size = prop_value * 0.01
+                        light_object.data.spot_size = prop_value * (pi / 180)
```
Comment (floxay) [inline src/mods/liana_main.py:1379]: use radians() from math

File: src/mods/liana_main.py (hunk 1 above)
Comment (KaiserM21) [reply src/mods/liana_main.py]: resolved

File: src/mods/liana_main.py (hunk 3 above)
Comment (KaiserM21) [reply src/mods/liana_main.py]: added elifs, dont know if too many

File: src/mods/liana_main.py (hunk 4 above)
Comment (KaiserM21) [reply src/mods/liana_main.py]: resolved

**Pull request merged**